tests/cypress/support/e2e.ts

```diff
@@ -76,6 +76,7 @@ declare global {
       createCloudCredential(cloudProvider: string, credentialName: string, accessKey: string, secretKey: string, region?: string): Chainable<Element>;
       createCloudCluster(cloudInstanceType: string, clusterName: string, subnetId: string): Chainable<Element>;
       deleteDownstreamCluster(clusterName: string, deleteOption?: boolean): Chainable<Element>;
+      addHelmOp(fleetNamespace: string?, repoName: string, repoUrl: string, chart: string, version?: string, values?: string, deployTo?: string, serviceAccountName?: string, targetNamespace?: string, helmAuth?: string): Chainable<Element>;
```

Comment:
`?` is typo ?
```suggestion
      addHelmOp(fleetNamespace: string, repoName: string, repoUrl: string, chart: string, version?: string, values?: string, deployTo?: string, serviceAccountName?: string, targetNamespace?: string, helmAuth?: string): Chainable<Element>;
```

Reply:
No, since it is defined as default to fleet-local I set it as optional to be added.